openprescribing/measure_definitions/edoxaban.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "Proportion of Direct Acting Oral Anticoagulants (DOACs) not prescribed as edoxaban",
+  "title": [
+    "Percentage DOACs items not prescribed as edoxaban"
+  ],
+  "description": [
+    "Percentage DOACs items not prescribed as edoxaban"
+  ],
+  "numerator_short": "DOAC items not prescribed as edoxaban",
+  "denominator_short": "DOACs items",
+  "why_it_matters": [
+    "In January 2022 the NHS in England <a href='https://www.england.nhs.uk/wp-content/uploads/2022/01/B1279-national-procurement-for-DOACs-commissioning-recommendations-v1.pdf'>",
```

Comment:
The new contract documents are out so I think we should update this before launch